yarn.lock

```diff
@@ -0,0 +1,10624 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Could you get rid of `yarn.lock` and make sure `package-lock.json` is up-to-date with the latest in package.json?

Reply:
The `package-lock.json` is up to date, and I removed the `yarn.lock`.